fe/fe-core/src/main/java/com/starrocks/load/streamload/StreamLoadMgr.java

```diff
@@ -181,19 +183,8 @@ public void beginLoadTaskFromBackend(String dbName, String tableName, String lab
     }
 
     // for sync stream load
-    public StreamLoadTask createLoadTask(Database db, String tableName, String label, String user, String clientIp,
-                                         long timeoutMillis, boolean isRoutineLoad, long warehouseId)
-            throws StarRocksException {
-        Table table;
-        Locker locker = new Locker();
-        locker.lockDatabase(db.getId(), LockType.READ);
-        try {
-            unprotectedCheckMeta(db, tableName);
-            table = GlobalStateMgr.getCurrentState().getLocalMetastore().getTable(db.getFullName(), tableName);
-        } finally {
-            locker.unLockDatabase(db.getId(), LockType.READ);
-        }
-
+    public StreamLoadTask createLoadTask(Database db, Table table, String label, String user, String clientIp,
```

Comment:
what's the difference of this interface compared to `createLoadTaskWithoutLock` after the code change.

Reply:
the `table` is after `checkMeta` before createLoadTask, I guess there is a reason that `checkMeta` should be called first.
`the` table in `createLoadTaskWithoutLock` is without check.